pkg/Directory.Build.props

```diff
@@ -24,7 +24,7 @@
     <PackageLicenseUrl>https://github.com/dotnet/machinelearning/blob/master/LICENSE</PackageLicenseUrl>
     <PackageProjectUrl>https://dot.net/ml</PackageProjectUrl>
     <PackageIconUrl>https://aka.ms/mlnetlogo</PackageIconUrl>
-    <PackageReleaseNotes>https://github.com/dotnet/machinelearning/tree/master/Documentation/release-notes</PackageReleaseNotes>
+    <PackageReleaseNotes>https://aka.ms/mlnetreleasenotes</PackageReleaseNotes>
```

Comment:
Who owns this path? Can you assign rights to edit it for ml.net group?

Reply:
Do we want a versioned link? http://aka.ms/mlnetreleasenotes-v0.2

If it's being built into the package itself, we may want a versioned link.

Reply:
The Aka.ms is pointing to the list of release notes, we can have a specific one when 1.0 drops

Reply:
We found on our team that if we have a versioned link, it gets stale very easily.  Especially as you start multiple servicing branches and shipping patches often.  See https://github.com/dotnet/corefx/issues/29314, https://github.com/dotnet/corefx/issues/26014. Someone needs to update the link for every preview/release/patch/etc.

So the current experiment is to have one URL that points to the "root" of the release notes. The users can navigate to the specific version as necessary.  It puts a little more work on the user, but at least it doesn't point them to the wrong place.

Reply:
> Who owns this path? Can you assign rights to edit it for ml.net group?

@shauheen created the aka.ms link for me.